uw-frame-components/portal/main.js

```diff
@@ -325,11 +325,11 @@ define([
         // https://github.com/Gillespie59/eslint-plugin-angular/issues/231
         // eslint-disable-next-line angular/on-watch
         $rootScope.$on('$routeChangeStart', function(event, next, current) {
-          var searchValue = '';
-          var paramToTackOn = '';
-          if(next.$$route && next.$$route.searchParam) {
-            paramToTackOn = APP_FLAGS.gaSearchParam || 'q';
-            searchValue = next.params[next.$$route.searchParam];
+          // eslint-disable-next-line angular/no-private-call
+          var nextRoute = next.$$route;
```

Comment:
Isn't next of type Route? Why is the internal route object needed?
https://docs.angularjs.org/api/ngRoute/service/$route#$routeChangeStart